modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -22,11 +24,20 @@ class Test{{classname}}(unittest.TestCase):
     def tearDown(self):
         pass
 
-    def test{{classname}}(self):
+    def make_instance(self, include_optional):
         """Test {{classname}}"""
-        # FIXME: construct object with mandatory attributes with example values
         # model = {{packageName}}.models.{{classFilename}}.{{classname}}()  # noqa: E501
-        pass
+        if (include_optional) :
+            return {{classname}}({{#vars}}
```

Comment:
Can you update this to:
```
            return {{classname}}(
{{#vars}}
                {{name}} = {{#example}}{{{.}}}{{/example}}{{^example}}None{{/example}}{{#hasMore}},{{/hasMore}}
{{/vars}})
            )
```
That way it is clearer that we are looping over vars
And we should be able to get rid of the # noqa: E501

Reply:
The generated code will have extra \n. Anyway, done.